Automattic/wp-calypso
# P2: signup: initial styling

Part of introducing the revamped signup flow for the P2 project.

In this PR, we add the redesigned "site" signup step for P2.

## Testing instructions

Navigate to `http://calypso.localhost:3000/start/p2` and go through the flow. It should look and work fine. Try both in an incognito window (=> new wp.com account) and with your current wp.com account.
